### DISCUSSION TITLE
Proposal - Change default Inter font to Atkinson Legible (or other)

### DISCUSSION THREAD

**Discussion by Draise14** [Bforartists/Bforartists#3875] [Ideas]
![image](https://github.com/Bforartists/Bforartists/assets/35357409/cf571c25-b01b-4ab1-b689-45cc8a62fe0b)
![image](https://github.com/Bforartists/Bforartists/assets/35357409/fa3becd2-d249-4cae-888c-8a85c0e31e2e)
![image](https://github.com/Bforartists/Bforartists/assets/35357409/c7a28086-be70-4a82-bdd4-37d2f6c2f503)
![image](https://github.com/Bforartists/Bforartists/assets/35357409/91cbf7db-21bf-4dc4-8d7e-8365cfb04f90)
![image](https://github.com/Bforartists/Bforartists/assets/35357409/9af40f45-4a49-48d8-8b89-a857a7a181cd)

In Blender 4.0+, the default font has changed. Right now the fonts are Inter, which is more compact and designed for GUI. But it also has some kerning issues which affects legibility specially at smaller scales. 

Some comments:
> Personal opinion. I absolutely hate Inter; it has a wide horizontal space not suitable for dense UIs. Other than that it's only decent for a generic sans typeface

> The kerning on the i And the n

someone pointed out a font designed for legibility called Atkinson Legible - which is also license compatible. I gave it a go. 

## Now:
Inter
![bforartists_cR2CtxcfDG](https://github.com/Bforartists/Bforartists/assets/35357409/79eebb26-9066-4a3b-be0a-21eff173a9ac)

## Proposed:
Atkinson Legible
**Note:** _The 0 is crossed out to differentiate between O_
https://brailleinstitute.org/freefont
This font was prized for low sight legibility and built by the same institute that made braille. It also is.. happenstance more compact. 
![bforartists_ojUMr9YbAl](https://github.com/Bforartists/Bforartists/assets/35357409/650aaa56-bad4-4db9-80e2-0cfc15f87898)

What do you think? To change default font is easy, but should we? A bit of branding? Focus on legibility? Or leave as is?

**Comment by Draise14** [comment]
The reason they chose a variable font inter is for variability though... so might not be a good idea

![image](https://github.com/Bforartists/Bforartists/assets/35357409/211fe0ac-9045-49d3-b098-e9195398732f)
https://projects.blender.org/blender/blender/pulls/112454

**Comment by ReinerBforartists** [comment]
Hmm, tough one. I like the Inter also better ^^

**Comment by Draise14** [comment]
Yeah, it's really minute the differences - other than the 0s. Probably a non-issue.
